assets/app/views/editor/edit-main.js

```diff
@@ -60,6 +63,27 @@ var EditView = Backbone.View.extend({
     this.pageSwitcher.set(childView);
 
     return this;
+  },
+  newPage: function(e) {
+    var self = this,
+        path = prompt('Please the new file path', 'pages/thing.md'),
+        opts = {
+          path: path,
+          content: '---\nyo: dude\nlayout: page\n---\n'
```

Comment:
Whoops, this is some debugging code, I'll go ahead and remove that.

The way I implemented the `defaults.yml` (which I think we should maybe to change to `_federalist_defaults.yml` and change `_navigation.json` to `_federalist_navigation.yml` so that they're consistent) is that it is a markdown document and the content of that document is just put into the new file being created. If there is no document, then nothing should be put into the page by default.

Any changes you think I should make to that process?